### ISSUE TITLE
vcSim property collector response does not have config.createDate

### ISSUE THREAD

**Issue by mridulganga** [vmware/govmomi#2280]
Hi I am trying to use vcsim for mocking vcenter api in some of my tests and I am having an issue related to createDate.
It seems that the config.CreateDate property is always unset for all the updateSet versions (including version 0)

I understand that vcSim is simplified version of vcenter apis and I cant expect it to have all the features, but createDate seems 
 like an essential field. Can we please get it in version 0 update set atleast?

ObjectUpdate -
```
(vmodl.query.PropertyCollector.ObjectUpdate) {
               dynamicType = <unset>,
               dynamicProperty = (vmodl.DynamicProperty) [],
               kind = 'enter',
               obj = 'vim.VirtualMachine:vm-66',
               changeSet = (vmodl.query.PropertyCollector.Change) [
                  (vmodl.query.PropertyCollector.Change) {
                     dynamicType = <unset>,
                     dynamicProperty = (vmodl.DynamicProperty) [],
                     name = 'name',
                     op = 'assign',
                     val = 'goimage6'
                  },
                  (vmodl.query.PropertyCollector.Change) {
                     dynamicType = <unset>,
                     dynamicProperty = (vmodl.DynamicProperty) [],
                     name = 'config.template',
                     op = 'assign',
                     val = true
                  },
                  (vmodl.query.PropertyCollector.Change) {
                     dynamicType = <unset>,
                     dynamicProperty = (vmodl.DynamicProperty) [],
                     name = 'summary.storage.committed',
                     op = 'assign',
                     val = 32
                  },
                  (vmodl.query.PropertyCollector.Change) {
                     dynamicType = <unset>,
                     dynamicProperty = (vmodl.DynamicProperty) [],
                     name = 'config.createDate',
                     op = 'assign',
                     val = <unset>
                  },
                  },
               ],
            }
         ],
      }
```
Note that config.CreateDate is unset

**Comment by mridulganga** [comment]
Thanks for quick response.

**Comment by dougm** [comment]
Looks like `CreateDate` was added in vSphere 6.7, vcsim was originally implemented against 6.5.  There are plenty of other fields vcsim doesn't populate, but we can add them as needed.
